enterprise_gateway/services/kernels/remotemanager.py

```diff
@@ -507,7 +533,6 @@ async def _launch_kernel(self, kernel_cmd, **kwargs):
 
         # Apply user_overrides to enable defaulting behavior from kernelspec.env stanza.  Note that we do this
         # BEFORE setting KERNEL_GATEWAY and removing {EG,KG}_AUTH_TOKEN so those operations cannot be overridden.
```

Comment:
This comment nolonger makes sense with the update statement removed.  Perhaps moving this following the setting of `KERNEL_GATEWAY`?
```suggestion
        # Remove {EG,KG}_AUTH_TOKEN so these envs cannot be overridden.
```